<launch>
	
	<node name="brand_descriptor_service" pkg="visual_descriptor" type="brand_service" />

	<node name="base_descriptor_service" pkg="visual_descriptor" type="base_service" />

	<node name="masked_base_descriptor_service" pkg="visual_descriptor" type="masked_base_service" />

	<!-- <node name="mfcc_descriptor_service" pkg="mfcc_descriptor" type="ros_service.py" /> -->
	
	<node name="v_bof" pkg="audiovisual_fusion" type="v_bof.py" output="screen" args="$(arg dataset)"/>
	  
</launch>